graphql-server/pkg/resolvers/proposal.go

```diff
@@ -21,6 +21,42 @@ func (r *proposalResolver) Type(ctx context.Context, obj *models.Proposal) (mode
 	return *proposalType, nil
 }
 
+func (r *proposalResolver) TallyResult(ctx context.Context, obj *models.Proposal) (*models.ProposalTallyResult, error) {
+	tally, err := pkgContext.GetDataLoadersFromCtx(ctx).Proposal.LoadProposalTallyResult(obj.ID)
+	if err != nil {
+		return nil, err
+	}
+	return tally, nil
+}
+
+func (r *proposalTallyResultResolver) Yes(ctx context.Context, obj *models.ProposalTallyResult) (string, error) {
```

Comment:
can we add alias type for BigInt -> string? for readability

Reply:
@chihimng 

https://github.com/oursky/likedao/blob/431786e9d3cb209f0be4322c62fc38940430ac2c/graphql-server/gqlgen.yml#L50-L53

It's actually defined, it's because we have defined ProposalTallyResult -> models.ProposalTallyResult and we are using `bunbig.Int` there. It seems like mapping `bunbig.Int -> string` doesn't work. Do you know if there's a way to map `bunbig.Int -> BigInt`?

Reply:
oh...

can we try https://github.com/xplorfin/gql-bigint ?

Reply:
bun requires a type to have `Scan` and `Value` functions to interact with the db and gqlgen requires the type to have `MarshalGQL` and `UnmarshalGQL` functions to not throw a resolver in your face :sosad: so we might need to make a type that handles both

Reply:
umm my concern is the resolver looks weird (to have string return type)... i think having bunbig.Int on db level & another bigint type on gql level is fine

Reply:
at least we can guard the returned value is really a big int, not arbitrary string